packages/docs/src/core/src/spec/transpilation-details.rst

```diff
@@ -87,6 +94,8 @@ To replace Call-type opcodes, we have to pass an existing slice of ``calldata``
 .. list-table::
     :widths: 50 50
     :header-rows: 1
+<<<<<<< HEAD
+=======
```

Comment:
This looks like an unresolved conflict. Choose one or the other and get rid of the metadata (<<<<<<< HEAD, ========, >>>>>>>).